db.py

```diff
@@ -91,6 +95,84 @@ async def get_botw_nomination(session, guild_id, member_id):
     return result[0] if result else None
 
 
+async def get_twitter_settings(session, guild_id):
+    statement = select(TwtSetting).where(TwtSetting._guild == guild_id)
+    result = (await session.execute(statement)).all()
+
+    return [r for (r,) in result]
```

Comment:
```suggestion
async def get_twitter_settings(session, guild_id):
    statement = select(TwtSetting).where(TwtSetting._guild == guild_id)
    result = (await session.execute(statement)).first()

    return result[0] if result else None
```

This also makes the multiple [0]'s above obsolete

Reply:
it do